packages/headless/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@sit-onyx/headless",
+  "description": "Headless Composables for Vue",
+  "version": "0.0.0",
+  "type": "module",
+  "author": "Schwarz IT KG",
+  "license": "Apache-2.0",
+  "exports": {
+    ".": {
+      "default": "./src/index.ts"
+    }
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "tsc --noEmit"
```

Comment:
This is rather a type-check command since it does not actually build anything, right?

We should also add it to the check workflow that runs for PRs

Reply:
Out-of-scope for this PR: I added a follow-up task: https://github.com/orgs/SchwarzIT/projects/2/views/1?pane=issue&itemId=48102251